src/test/java/org/alfresco/repo/web/scripts/servlet/RemoteAuthenticatorFactoryAdminConsoleAccessTest.java

```diff
@@ -198,6 +212,35 @@ public void testAdminCanAccessAdminConsoleHelperScript()
         complexCheckOfScriptCases(families);
     }
 
+    /**
+     * Tested access to the AdminConsole for an non literal admin user
+     * but with admin permissions (user added to ALFRESCO_ADMINISTRATORS)
+     * and accessing via Basic Auth
+     */
+    @Test
+    public void testUserCanAccessAdminConsoleScript()
+    {
+        Set<String> families = new HashSet<>();
+        families.add("AdminConsole");
+
+        // Run as System to be able to give permissions
+        this.authenticationComponent.setSystemUserAsCurrentUser();
+
+        String username = RandomStringUtils.randomAlphabetic(10);
+        String password = RandomStringUtils.randomAlphabetic(10);
+        this.createUser(username, password);
+        this.authorityService.addAuthority("GROUP_ALFRESCO_ADMINISTRATORS", username);
+
+        this.authenticationComponent.clearCurrentSecurityContext();
+
+        String headerToAdd = this.getBasicAuthHeader(username, password);
+
+        checkAdminConsoleFamilyWithBasicAuthHeaderPresentUser(families, headerToAdd);
+
+        this.deleteUser(username);
```

Comment:
If the user is random, there is no need to delete it. Yes, the number of users will grow if you run the test many times, but it will slow down each test execution for cleanup. More over the cleanup will not be run if the test fails.

Reply:
I thought the first part of your comment. But apparently it's not necessary. Will do.